www/src/components/PropsTable.jsx

```diff
@@ -28,7 +28,7 @@ const Prop = ({
   <li className="px-4 border-top border-light-300">
     <div className="my-3">
       <div className="mb-2">
-        <h5 className="d-inline mb-0 mr-1">{`${name} `}</h5>
+        <h1 className="d-inline mb-0 mr-1">{`${name} `}</h1>
```

Comment:
[curious] There is already an `h1` on the page, in the "Paragon Design System" title. As is, each prop name is going to be an `h1`, which generally looks rather large.

<img width="983" alt="image" src="https://user-images.githubusercontent.com/2828721/171841840-794ca566-64d6-49ed-8cde-5149a32370b3.png">

Actually, instead of a heading tag at all, I think it might be more appropriate for this to be written as:

```jsx
<span class="mr-2 font-weight-bold">{name}</span>
```

as I don't believe these should be treated as headings semantically.